assets/app/views/browse/build.html

```diff
@@ -100,6 +100,10 @@
                         run="logCanRun">
                       </log-viewer>
                     </uib-tab>
+                    <uib-tab active="selectedTab.events">
+                      <uib-tab-heading>Events</uib-tab-heading>
+                      <events resource-kind="Pod" resource-name="{{$parent.build.metadata.annotations['openshift.io/build.pod-name']}}" project-context="$parent.projectContext" ng-if="selectedTab.events"></events>
```

Comment:
I don't think you need the `$parent.` before these.

Use the annotation filter.

```
resource-name="{{build | annotation : 'buildPod'}}"
```